e2e/history_components.test.js

```diff
@@ -0,0 +1,29 @@
+const { log } = require('detox');
+
+describe('Example', () => {
+  beforeAll(async () => {
+    await device.launchApp();
+  });
+
+  beforeEach(async () => {
+    // jest.useFakeTimers();
+    await device.reloadReactNative();
+  });
+  
+  // afterEach(() => {
+  //   jest.runOnlyPendingTimers()
+  //   jest.useRealTimers()
+  // })
+  
+  it('loads my test wallet', async () => {
+    await element(by.id("loadingapp.restorewalletseed")).tap();
+    await element(by.id("seed.seedplaceholder")).replaceText('lottery multiply patient simple ivory leisure swift square west despair beauty match crowd margin reject box always title photo remind word diet ecology badge');
+    await element(by.id("birthdayinput")).replaceText('1994000');
+    await element(by.text("RESTORE WALLET")).tap();
+  });
+  
+  it('syncs a transaction in 100000ms', async () => {
+    await waitFor(element(by.id("transactionList.1"))).toBeVisible().withTimeout(100000)
+    await element(by.id("transactionList.1")).tap();
+  });
+});
```

Comment:
This is pretty cool!  I think we should use a standard test SEED.   We have some in zingolib.

Reply:
made it... can we add this to our standard test seed library? i already funded it

Reply:
@fluidvanadium IMO here can be really really useful to have a lot of comments explaining for what is this test, and its parts....